client/user/src/mealorder/meal-order-main-app.html

```diff
@@ -0,0 +1,357 @@
+<link rel="import" href="../../bower_components/iron-icons/maps-icons.html">
+
+<link rel="import" href="shell/redux-store.html">
+<dom-module id="meal-order-main-app">
+    <template>
+        <style>
+            :host {
+                font-family: 'Roboto', 'Noto', sans-serif;
+                --app-primary-color: #4285f4;
+                --app-secondary-color: black;
+                display: block;
+            }
+
+            app-header {
+                background-color: #616161;
+                color: #fff;
+                right: 300px;
+            }
+
+            app-header .main-title {
+                font-size: 1.375em;
+            }
+
+            app-drawer-layout {
+                background-color: #F2F2F2;
+            }
+
+            app-drawer-layout:not([narrow]) [drawer-toggle] {
+                display: none;
+            }
+
+            app-header app-toolbar .container {
+                display: flex;
+                width: 100%;
+                align-items: center;
+                justify-content: space-between;
+            }
+
+            app-header app-toolbar .container .order-container {
+                display: flex;
+            }
+
+            app-header app-toolbar .container .date-container {
+                display: flex;
+                margin-left: 10px;
+                align-items: center;
+            }
+
+            app-header .history-of-orders a {
+                font-size: 0.8em;
+                text-decoration: none;
+                color: white;
+            }
+
+            app-header .history-of-orders a:hover {
+                color: #179EFF;
+            }
+
+            app-header .history-icon {
+                display: none;
+                color: white;
+            }
+
+            app-header .calendar {
+                font-weight: lighter;
+                font-size: 0.75em;
+            }
+
+            app-header .order {
+                font-size: 0.75em;
+            }
+
+            .order-date {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+                width: 100%
+            }
+
+            .order-date .order {
+                font-size: 24px;
+            }
+
+            .order-date .day {
+                font-size: 0.7em;
+                font-weight: lighter;
+                text-align: center;
+            }
+
+            .order-date .day-of-the-week {
+                font-size: 22px;
+            }
+
+            @media only screen and (max-width: 600px) {
+
+                .history-of-orders {
+                    display: none;
+                }
+
+                app-header .history-icon {
+                    display: block;
+                }
+
+                app-header .date-container .date-range-icon {
+                    display: none;
+                }
+
+                app-header .date-container .calendar {
```

Comment:
Why not to use ids instead of class names for single items?

Reply:
`meal-order` and `menu order` are described in `#53.